static/katex.less

```diff
@@ -372,6 +372,26 @@
         }
     }
 
+
+
+    .overrightarrow {
+        .overrightarrow-line {
+            width: 100%;
+            height: 1em;
+            position: relative;
+            overflow: hidden;
+
+            &:before {
+                content: "−−−−−−−−−−−−−−−−−→";
```

Comment:
This is a clever solution, but it fails if the math under the arrow is too wide.  Could you add a comment with the limitations of this approach and a TODO to make this better at some point?  The limitations that I see here are that this won't work with `\overleftrightarrow` and if the expression under the arrow is too large it won't extend far enough.  Also, do we need both .overrightarrow and .overrightarrow-line?  Not sure how many people would use `\overleftarrow`, but it seems like an easy addition.